entrypoint/secretsencrypt/encryption_suite_test.go

```diff
@@ -0,0 +1,62 @@
+package secretsencrypt
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/config"
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var cfg *config.Product
+
+func TestMain(m *testing.M) {
+	var err error
+	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
+	flag.Parse()
+
+	cfg, err = shared.EnvConfig()
+	if err != nil {
+		return
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestSecretsEncryptionSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Secrets Encryption Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+	if err := config.SetEnv(shared.BasePath() + fmt.Sprintf("/config/%s.tfvars", cfg.Product)); err != nil {
+		Expect(err).To(BeNil(), fmt.Sprintf("error loading tf vars: %v\n", err))
+	}
+	Expect(os.Getenv("server_flags")).To(ContainSubstring("secrets-encryption:"),
+		"FATAL: Add secrets-encryption:true to server_flags for this test")
```

Comment:
Can we use log-level here instead of printing `FATAL...`?

Reply:
This message is within Expect - it is not allowing to add an extra shared.LogLevel here. So changed from FATAL to Error. Hope this is okay?